files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md

```diff
@@ -67,22 +67,20 @@ function draw() {
 draw();
 ```
 
-## 移动 Translating
+## 移动
 
-![](canvas_grid_translate.png)
-
-我们先介绍 `translate`方法，它用来移动 canvas 和它的原点到一个不同的位置。
+我们先介绍 `translate` 方法，它用来移动 canvas 和它的原点到一个不同的位置。
 
 - `translate(x, y)`
   - : `translate`方法接受两个参数。*x *是左右偏移量，_y_ 是上下偏移量，如右图所示。
 
```

Comment:
canvas_grid_translate.png has not removed, it moved to here

Reply:
```suggestion

![画布从网格上的原点水平向下向右平移“x”单位，垂直移动“y”单位。](canvas_grid_translate.png)

```

Reply:
This has been remove from the upstream PR by mistake. But I forgot to add it back when the upstream has fixed this. Thanks for the reminder